README.md

```diff
@@ -42,8 +42,11 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
-
+S -> C
+S -> SCS
+C -> "r"
+C -> "g"
+C -> "b"
```

Comment:
# 0

Try running this in your head, using it to generate a path of length 3. You'd need to execute 

    S -> S C S

The first S on the RHS will have to expand to C, which expands to a color. Let's say "r".
The C will expand to a color, say "g". The final S must expand to C, which expands to (say) "g". So we have "rgb", which isn't valid.

Or, we could try again, but have the last S expand itself to SCS. That would lead to "rgrgb".

I was looking for something like

       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b